include/cameras/CameraH264.h

```diff
@@ -44,9 +44,13 @@ class CameraH264 : public CameraBase
 
 private:
   constexpr static int DEFAULT_BITRATE = 8000000;
+  constexpr static int DEFAULT_PROFILE = 0;
+  constexpr static int DEFAULT_LEVEL = 0;
```

Comment:
Level is of type [NvMediaEncodeLevel](https://developer.nvidia.com/docs/drive/drive-os/latest/linux/sdk/api_reference/group__x__image__encode__api.html#ga36340ae004bfbd7ac769473a8797e374).

Profile is of type [NvMediaEncodeProfile](https://developer.nvidia.com/docs/drive/drive-os/latest/linux/sdk/api_reference/group__x__image__encode__api.html#gaf69cad0f999c9b6c5dd854da534ecbb7).